### PR TITLE
Quay Enterprise: app token expiry

### PR DESCRIPTION
**Pull request by ibazulic**
Adds an explanation  about app tokens, icons in the app token list and expiry.

### CODE REVIEW

File: quay-enterprise/app-tokens.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Quay Enterprise Application Tokens #
+
+Since Quay Enterprise version 2.7.0, a new way of authenticating to Quay from Docker CLI (and other apps) was added, in the form of application tokens. These tokens don't expire by default, but this expiration can be set inside the Quay superadmin control panel. 
+To enable expiration of application tokens, the following option must be selected in the Quay Enterprise suerpadmin control panel:
```
Comment (zbwright) [inline quay-enterprise/app-tokens.md]: ... superadmin ...

File: quay-enterprise/app-tokens.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+![Quay Enterprise superadmin control panel](https://github.com/ibazulic/docs/blob/master/quay-enterprise/img/set-token-expiration-time.png "Quay Enteprrise admin control panel")
+
+Note that the number of days **needs** to be specified, if the text field is blank, the tokens will never expire.
+
+## Icons explained ##
+
+In a typical situation, a repo owner might have multiple tokens created for multiple applciations, and every token has its own expiry time, as we can see here:
```
Comment (zbwright) [inline quay-enterprise/app-tokens.md]: In most cases, a repo owner will have multiple tokens for multiple applications, and each token will have its own expiry time.

File: quay-enterprise/app-tokens.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+![Application Tokens](https://github.com/ibazulic/docs/blob/master/quay-enterprise/img/app-token-list.png "Application tokens")
+
+Every token is marked with a triangle that shows the current state of the token:
+
+* red triangle (not shown): marks the expired token
+* yellow triangle: tokens that will expire in 30 days time.
+* green triangle: token that has more than 30 days left till expiration.
+
+Note that currently a repo owner **can't** choose the token expiration time. This can be set only by the Quay admin.
```
Comment (josephschorr) [inline quay-enterprise/app-tokens.md]: `repo owners` -> `namespace owners`

**Comment by ibazulic** [comment]
updated the file and corrected mistakes.

File: quay-enterprise/app-tokens.md [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
+To enable expiration of application tokens, the following option must be selected:
 
 ![Quay Enterprise superadmin control panel](https://github.com/ibazulic/docs/blob/master/quay-enterprise/img/set-token-expiration-time.png "Quay Enteprrise admin control panel")
 
 Note that the number of days **needs** to be specified, if the text field is blank, the tokens will never expire.
 
 ## Icons explained ##
 
-In a typical situation, a repo owner might have multiple tokens created for multiple applciations, and every token has its own expiry time, as we can see here:
+In most cases, a repo owner will have multiple tokens for multiple applications and each token will have its own epiry time.
```
Comment (josephschorr) [inline quay-enterprise/app-tokens.md]: Don't use `repo owner`. These are `namespace owners`. Also, `expiry` is misspelled, and `expiration` is preferred

**Comment by ibazulic** [comment]
updated the main document and made some additional changes regarding grammar.

**Comment by kbrwn** [comment]
@josephschorr is this good to merge?

**Comment by josephschorr** [comment]
@kbrwn Icons screenshot and text needs to be updated to reflect the bug fixes

**Pull request closed without merging**